ecms_base/ecms_base.install

```diff
@@ -2387,3 +2387,16 @@ function ecms_base_update_10212(array &$sandbox): void {
     $active_storage->write("{$config}", $install_source->read("{$config}"));
   }
 }
+
+/**
+ * Install drupal/password_policy, ultimate_cron module.
+ *
+ * Runs as part of 1.1.5 tag.
+ */
+function ecms_base_update_10213(array &$sandbox): void {
+  $modules_to_install = [
+    'password_policy',
+    'ultimate_cron',
```

Comment:
Does ulitmate cron have any configuration settings that are not the default that should be installed?